lte/gateway/configs/sctpd.yml

```diff
@@ -0,0 +1,14 @@
+---
+#
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+log_level: INFO
```

Comment:
since this will take precedence over orc8r streamed configs, let's comment it out and place a comment on top stating that either it can be commented out here or an override config can be specified under `/var/opt/magma/configs/sctpd.yml`.